cupy/manipulation/rearrange.py

```diff
@@ -91,4 +118,44 @@ def roll(a, shift, axis=None):
         res[r_ind2] = a[ind2]
         return res
 
-# TODO(okuta): Implement rot90
+
+def rot90(a, k=1, axes=(0, 1)):
+    """Rotate an array by 90 degrees in the plane specified by axes.
+
+    Args:
+        a (~cupy.ndarray): Array of two or more dimensions.
+        k (int): Number of times the array is rotated by 90 degrees.
+        axes: (tuple of ints): The array is rotated in the plane defined by
+            the axes. Axes must be different.
+
+    Returns:
+        ~cupy.ndarray: Output array.
+
+    .. seealso:: :func:`numpy.rot90`
+    """
```

Comment:
It would be better to describe remarks that this document is just copied or altered from the original one.
See: https://github.com/pfnet/chainer/blob/master/docs/source/contribution.rst#coding-guidelines